airflow/providers/google/cloud/transfers/sftp_to_gcs.py

```diff
@@ -85,13 +89,16 @@ def __init__(
         source_path: str,
         destination_bucket: str,
         destination_path: str | None = None,
-        gcp_conn_id: str = "google_cloud_default",
         sftp_conn_id: str = "ssh_default",
+        gcp_conn_id: str = "google_cloud_default",
         mime_type: str = "application/octet-stream",
         gzip: bool = False,
         move_object: bool = False,
         impersonation_chain: str | Sequence[str] | None = None,
         sftp_prefetch: bool = True,
+        use_stream: bool = False,
+        log_stream_progress: bool = False,
+        stream_chunk_size = 65536,
```

Comment:
I would recommend the default stream chunk to be 32768, since this is the Paramiko SFTPFile.MAX_REQUEST_SIZE